.buildkite/pipelines/quality-gates/pipeline.tests-qa.yaml

```diff
@@ -47,8 +47,14 @@ steps:
 
   - wait: ~
 
-  - label: ":judge::seedling: Trigger Manual Tests Phase"
+  - label: ":judge::seedling: Trigger Manual Tests Phase - Kibana Release Manager"
```

Comment:
Is there any other way to change the name displayed in Buildkite?

<img width="385" alt="image" src="https://github.com/elastic/kibana/assets/5831975/1b2c932c-6daf-4f24-b7c7-9ecfd0fbff8d">

Reply:
Not really, but maybe I'm misunderstanding.  What sort of change are you looking for?

Reply:
@ramonbutter do you know?

Reply:
Ramon's on PTO. What kind of flexibility are you looking for?

Reply:
The way you've updated the label should update the name of the manual trigger. It looks like `Manual Verification Required` is currently hard-coded in the manual-verification-agent. It could certainly be parameterized, but will that be needed if the trigger label is updated? I think this is the only scenario I know of where teams have been specifying more than one manual test phase.

Reply:
Thanks, I think that this should be enough in this case, let's see it in practice.

Reply:
In this comment:
https://elastic.slack.com/archives/C05NJL80DB8/p1694440085073759
I highlighted the problematic when collecting several manual approval in a single pipeline directly.
TLDR setting the permissions correctly for each individual manual gate is not as trivial.
Some miss clicks will happens where team members approve other teams gates.
When we are going with the suggestion approach here, we are accepting this as an ok state.
But yes otherwise should be not too complicated to add some support for changing the lock and notification messages.

Reply:
would this fit your needs?:
https://github.com/elastic/ci-agent-images/pull/399

Reply:
> In this comment: https://elastic.slack.com/archives/C05NJL80DB8/p1694440085073759 I highlighted the problematic when collecting several manual approval in a single pipeline directly. TLDR setting the permissions correctly for each individual manual gate is not as trivial. Some miss clicks will happens where team members approve other teams gates. When we are going with the suggestion approach here, we are accepting this as an ok state. But yes otherwise should be not too complicated to add some support for changing the lock and notification messages.

I'll leave this to the Kibana operations team (see https://github.com/elastic/kibana/pull/167502#discussion_r1341042474).



> would this fit your needs?: [elastic/ci-agent-images#399](https://github.com/elastic/ci-agent-images/pull/399)

Looks great thanks!

Reply:
please review my PR if this is sufficient

Reply:
@cachedout WDYT?